src/components/Hero.tsx

```diff
@@ -0,0 +1,118 @@
+          <img src={searchNormal} alt="search icon" />
+          <input
+            onFocus={() => setFocus(true)}
+            onBlur={() =>
+              setTimeout(() => {
+                setFocus(false);
+              }, 200)
+            }
+            onChange={(e => setSearchTerm(e.target.value))}
+            type="text" placeholder='Search repositories'
+            className='w-full outline-none text-[16px] text-lightSlate'
+          />
+          {/* todo: implement keyboard shortcut to bring the input field in focus - issue #205 */}
+          {/* <img className='pt-[7px]' src={cmdK} alt="command k" /> */}
+        </div>
+        <div className='mt-[10px] flex w-full justify-center relative'>
+          { results.length > 0 && hasFocus && 
+            <div className='flex md:min-w-[400px] pb-[8px] absolute z-50 max-w-[400px] flex-col bg-white rounded-[10px]'>
+              <div className='bg-gray-100 py-[10px] px-[10px] md:px-[15px] border-b-gray-100 border-b-[2px] rounded-[10px] rounded-b-none w-full'>
+                <p className='text-gray-500 text-[14px] font-semibold'>Repository</p>
+              </div>
+
+              {
+                results.map( ({full_name, description, issues, stars, user_id }) => (
+                  <a
+                  key={user_id}
+                  href={`https://app.opensauced.pizza/repos/${full_name}`}
+                  rel="noreferrer"
+                  target="_blank"
+                  >
+                    <div  className='flex flex-col hover:bg-gray-50 '>
+                      <div className='flex flex-col px-[10px] md:px-[15px] py-[10px]'>
+                        <div className='flex items-center gap-x-[10px] mb-[5px]'>
+                          <div className='w-[25px] h-[25px] overflow-hidden border-gray-400 border-[1px] bg-red-100  rounded-full'>
+                            <img className='w-full h-full' src={`https://avatars.githubusercontent.com/u/${user_id}`} alt="" />
+                          </div>
+
+                          <p className='text-[16px] text-gray-500 font-semibold'>{full_name}</p>
+                        </div>
+
+                        <p className='text-[14px] text-gray-500'>{description}</p>
+
+                        <div className='flex justify-between mt-[8px]'>
+                          <div className='flex gap-x-[5px]'>
+                            <div className='w-[20px] h-[20px] rounded-full'>
+                              {/* todos: add contributors avator here */}
+                            </div>
+                            
+                          </div>
+                          <div className='flex gap-x-[6px]'>
+                            <div className='flex items-center gap-x-[5px]'>
+                              <img src={issueIcon} alt="issue icon" />
```

Comment:
Are the icons here actionable? If not maybe we should change the img tags; that would get rid of the alt issue as well.

Same thing for the star icon

Reply:
@0-vortex  This image tag with icons is representing issues and stars. If we remove this icon, the issue number and star number would make any sense to the user?

Reply:
Not remove it completely, change the img tag or add a proper alt text similar to the above.

Generally we don't use img tags to describe css sprites in html but for the purpose of fast forwarding the visual features, an alt text will do just fine!